### PR TITLE
Enhances snackbar

### LINKED ISSUE #7478: [UI] Add hyperlink in snackbar to open Notification Center
#### Current Behavior
When a user sees a snackbar with a summary of what is otherwise detailed information, they are intrigued to click to find out more information. Currently, the snackbar for Kubernetes component registratio is not clickable which leaves the user hunting for where to find additional details.

Screenshot 2023-04-04 at 9 20 03 PM

#### Desired Behavior
Ideally, the user can click on the snack bar to open up the respective message inside of notification Center

#### Acceptance Tests
- Users can click on the notification causing the Notification Center to open and the corresponding notification to be expanded and in focus.

#### Mockups
The notification highlighted in red should be expanded and in focus on click of the snackbar.

Screenshot 2023-04-04 at 9 20 36 PM

---

#### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)

- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🖥 [Contributing to Meshery UI](https://docs.meshery.io/project/contributing/contributing-ui)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by mdsahiloss**
**Notes for Reviewers**

This PR fixes #7478

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by MdSahil-oss** [comment]
Hi @leecalcote, I've added an icon-button instead in snackbar instead of making the whole snackbar clickable. Please review it, If it's fine then I'll move further on this enhancement :)

**Comment by Abhishek-kumar09** [comment]
@MdSahil-oss Could you attach a screenshot or video demo?

**Review by Abhishek-kumar09 — COMMENTED** [review]
Just tried on my local machine. The changes so far are good. One suggestion I would be having is highlight the expanded notification temprararily. example is stackoverflow, where when you share an answer the answer got some temporary highlight effect and gets normal afterwards.

**Comment by MdSahil-oss** [comment]
Thanks for review @Abhishek-kumar09, Okay! I'll highlight the selected event for a while in the notification center, Afterwards will add the complete working video :)

**Comment by MdSahil-oss** [comment]
Hi @Abhishek-kumar09, I've updated the PR for highlighting feature, Here is the working video :-
[Screencast from 11-04-23 05:39:20 PM IST.webm](https://user-images.githubusercontent.com/85174511/231160715-7f88977d-27f6-4d7f-91e4-f0838beb4175.webm)

File: ui/components/MesherySnackbarWrapper.js [hunk 1]
```diff
@@ -128,18 +131,26 @@ function MesherySnackbarWrapper(props) {
   const [socialExpand, setSocialExpand] = useState(false);
   const [anchorEl, setAnchorEl] = useState(null);
   const [socialMessage, setSocialMessage] = useState("");
+  const [highlight, setHighlight] = useState(false)
 
   const handleExpandClick = () => {
     setExpanded(!expanded);
+    // expand && (expand = false);
```
Comment (Abhishek-kumar09) [inline ui/components/MesherySnackbarWrapper.js:138]: lets delete the commented code if it not providing any help.

**Review by Abhishek-kumar09 — DISMISSED** [review]

File: ui/components/MesherySnackbarWrapper.js (hunk 1 above)
Comment (MdSahil-oss) [reply ui/components/MesherySnackbarWrapper.js]: Done 👍🏽

**Review by Abhishek-kumar09 — DISMISSED** [review]

File: provider-ui/package-lock.json [hunk 2]
```diff
@@ -11073,7 +11072,6 @@
       "resolved": "https://registry.npmjs.org/@cypress/webpack-preprocessor/-/webpack-preprocessor-5.16.1.tgz",
       "integrity": "sha512-KZ6xdQmPd0YocbxDJc7l3Ov53eYmkWLCkpbMQSVIWSTZMjzK96V8GG7tGSg9TGUEeYCZitDli8NijvoXCRJU7g==",
       "requires": {
-        "@babel/core": "^7.0.1",
```
[the remark this answers is not in the document]
Comment (MdSahil-oss) [reply provider-ui/package-lock.json:11076]: I don't think so, let me correct it :)

[the remark this answers is not in the document]
Comment (MdSahil-oss) [reply provider-ui/package-lock.json]: @leecalcote I've removed such accidental changes, I hope now PR is good :)

**Review by leecalcote — APPROVED** [review]
Looking better. There are couple of other items to address 👍

File: ui/components/MesherySnackbarWrapper.js [hunk 3]
```diff
@@ -62,6 +62,9 @@ const styles = (theme) => ({
     alignItems : "center",
   },
   snackbarContent : { [theme.breakpoints.up("sm")] : { minWidth : "344px !important", }, },
+  snackbarContentBorder : {
+    border : "1px solid rgba(102, 102, 102, 1)"
```
Comment (leecalcote) [inline ui/components/MesherySnackbarWrapper.js:66]: This color isn't available in the theme?

File: ui/components/MesheryNotification.js [hunk 4]
```diff
@@ -161,20 +163,30 @@ class MesheryNotification extends React.Component {
...
-          <CloseIcon style={iconMedium} />
-        </IconButton>
+        <>
+          <IconButton
+            key="eye"
+            aria-label="eye"
+            color="inherit"
+            onClick={() => self.openEventInNotificationCenter(operation_id)}
+          >
+            <VisibilityIcon style={iconMedium} />
```
Comment (leecalcote) [inline ui/components/MesheryNotification.js:179]: @MdSahil-oss You're going to change to `MdInfoOutline`, right?

**Review by leecalcote — CHANGES_REQUESTED** [review]

File: ui/components/MesherySnackbarWrapper.js (hunk 3 above)
Comment (MdSahil-oss) [reply ui/components/MesherySnackbarWrapper.js:66]: Not available in the theme, But It is used as `iconColor` at line number 50 in the same file.

File: ui/components/MesheryNotification.js (hunk 4 above)
Comment (MdSahil-oss) [reply ui/components/MesheryNotification.js]: @leecalcote Now, I've replaced `VisibilityIcon` with `InfoIcon` :)

**Pull request merged**